### ISSUE TITLE
Bug_0

### ISSUE THREAD

**Issue by Bhaney44** [Bhaney44/AlgorandDevelopment#1]
I'm working my way from running a node to creating an ASA on the Algorand Network, I am anticipating using the mainet. I saw running your own node takes a few days to catch up, but I am not sure at all what that means, relative to using docker or a third-party for a shorter setup time. I am still getting errors when I check the status for the node. For example, "-bash: goal: command not found" which results from running: "goal node status" from the command line in a virtual env, in the node directory. I can physically see the goal file on my machine, but I am not sure what it is or how to work with it. I'm still hacking away on this like a total noob under the Start Building tab in the Algorand developer portal and would sincerely appreciate any advice or suggestions.

As far as my troubleshooting so far, I read through the docs in the [developer portal](https://developer.algorand.org/docs).  Because I was still having trouble with the command line syntax and interacting with the node files I read through a good [review](https://www.davidbaumgold.com/tutorials/command-line/#mac-os-x) on command line syntax for MacOS.

**Comment by Bhaney44** [comment]
**Update**

I tried creating a new virtual environment and re-installing from scratch. However, there is still an error in finding the files downloaded to the machine. This is most likely a pathing problem. The following code maps the command line syntax I used, the error read out, an alternative, and another error read out. 

```
//// Install on Mac
// cd node
// curl https://raw.githubusercontent.com/algorand/go-algorand-doc/master/downloads/installers/update.sh -O
// chmod 544 update.sh
// ./update.sh -i -c stable -p ~/node -d ~/node/data -n

//// Installing the Devtools
// If algorand has not been previously installed, it will automatically download it.
// If algorand is installed but older than the devtools, it will automatically upgrade it.

//// Start node
// sudo systemct1 start algorand
// ERROR: sudo: systemct1: command not found

// goal node status -d /var/lib/algorand
// ERROR: -bash: goal: command not found
```

Essentially the problem across the board is that the machine cannot find the commands. Next step is to find a way to create a clear path forward, to allow the commands to be appropriately identified and executed.

**Comment by Bhaney44** [comment]
**Update**

I did some work on path and command line syntax to understand the problem better. I created a repository, [NodeInstall](https://github.com/Bhaney44/AlgorandDevelopment/tree/main/NodeInstall) dedicated to working on this problem and developing command line syntax for installing a node. The repository focuses on the following the Algorand install instructions, the command not found error, and pathing.
